contracts/adapters/FeeSplitExtension.sol

```diff
@@ -52,92 +58,127 @@ contract FeeSplitAdapter is BaseAdapter, TimeLockUpgrade {
 
+    // Address which receives operator's share of fees when they're distributed. (See IIP-72)
+    address public operatorFeeRecipient;
+
     /* ============ Constructor ============ */
 
     constructor(
         IBaseManager _manager,
         IStreamingFeeModule _streamingFeeModule,
         IIssuanceModule _issuanceModule,
-        uint256 _operatorFeeSplit
+        uint256 _operatorFeeSplit,
+        address _operatorFeeRecipient
     )
         public
-        BaseAdapter(_manager)
+        BaseExtension(_manager)
     {
         streamingFeeModule = _streamingFeeModule;
         issuanceModule = _issuanceModule;
         operatorFeeSplit = _operatorFeeSplit;
+        operatorFeeRecipient = _operatorFeeRecipient;
         setToken = manager.setToken();
     }
 
     /* ============ External Functions ============ */
 
     /**
      * ANYONE CALLABLE: Accrues fees from streaming fee module. Gets resulting balance after fee accrual, calculates fees for
-     * operator and methodologist, and sends to each. NOTE: mint/redeem fees will automatically be sent to this address so reading
-     * the balance of the SetToken in the contract after accrual is sufficient for accounting for all collected fees.
+     * operator and methodologist, and sends to operator and methodologist, and sends to operatorFeeRecipient and methodlogist
+     * respectively. NOTE: mint/redeem fees will automatically be sent to this address so reading the balance of the SetToken
+     * in the contract after accrual is sufficient for accounting for all collected fees.
      */
     function accrueFeesAndDistribute() public {
+        // Emits a FeeActualized event
         streamingFeeModule.accrueFee(setToken);
-        
-        uint256 totalFees = setToken.balanceOf(address(manager));
-        
-        address operator = manager.operator();
+
+        uint256 totalFees = setToken.balanceOf(address(this));
+
         address methodologist = manager.methodologist();
 
         uint256 operatorTake = totalFees.preciseMul(operatorFeeSplit);
         uint256 methodologistTake = totalFees.sub(operatorTake);
 
         if (operatorTake > 0) {
-            invokeManagerTransfer(address(setToken), operator, operatorTake);
+            setToken.transfer(operatorFeeRecipient, operatorTake);
         }
 
         if (methodologistTake > 0) {
-            invokeManagerTransfer(address(setToken), methodologist, methodologistTake);
+            setToken.transfer(methodologist, methodologistTake);
         }
 
-        emit FeesAccrued(operator, methodologist, operatorTake, methodologistTake);
+        emit FeesDistributed(operatorFeeRecipient, methodologist, operatorTake, methodologistTake);
     }
 
     /**
-     * ONLY OPERATOR: Updates streaming fee on StreamingFeeModule. NOTE: This will accrue streaming fees though not send to operator
-     * and methodologist.
+     * MUTUAL UPGRADE: Updates streaming fee on StreamingFeeModule. NOTE: This will accrue streaming
```

Comment:
Maybe we should make a note for all of these `mutualUpgrade` and `timeLockUpgrade` combo functions that it will require each party submitting two txns. One to set the timelock and the next to execute the function

Reply:
f19d7ea